builtin/providers/aws/resource_aws_elb.go

```diff
@@ -300,6 +306,18 @@ func resourceAwsElbRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("security_groups", lb.SecurityGroups)
 	if lb.SourceSecurityGroup != nil {
 		d.Set("source_security_group", lb.SourceSecurityGroup.GroupName)
+
+		// Manually look up the ELB Security Group ID, since it's not provided
+		var elbVpc string
+		if lb.VPCId != nil {
+			elbVpc = *lb.VPCId
+		}
+		sgId, err := sourceSGIdByName(meta, *lb.SourceSecurityGroup.GroupName, elbVpc)
+		if err != nil {
+			log.Printf("[WARN] Error looking up ELB Security Group ID: %s", err)
```

Comment:
Similar to a Q I asked elsewhere today - if there was an error, here, isn't that a big enough deal to stop? Perhaps the answer is "no" but I figure it's worth considering.